src/tools/webrtc-test-client/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
could you create a gn-based build instead? Or at least support it in build_examples.py?

We would like to avoid the proliferation of custom scripts as generally we have a lot of targets already.

Reply:
Yes, we are working on integrating the build process to follow Matter build flow.

Made a PR to just share our current status in the call.